src/matrix/kernelkroneckermat.jl

```diff
@@ -25,3 +25,35 @@ end
     k(x,x') = ∏ᵢᴰ k(xᵢ,x'ᵢ)
 """
 @inline iskroncompatible(κ::Kernel) = false # Default return for kernels
+
+@inline ismatrixkroncompatible(κ::MOKernel) = false # Default return for kernels
+@inline ismatrixkroncompatible(κ::IndependentMOKernel) = true
+@inline ismatrixkroncompatible(κ::IntrinsicCoregionMOKernel) = true
+
+function _kroneckerkernelmatrix(Ktmp, B, ::MOInputIsotopicByFeatures)
+    return Kronecker.kronecker(Ktmp, B)
+end
+
+function _kroneckerkernelmatrix(K, Ktmp, B, ::MOInputIsotopicByOutputs)
```

Comment:
The `K` seems like it shouldn't be here:
```suggestion
function _kroneckerkernelmatrix(Ktmp, B, ::MOInputIsotopicByOutputs)
```

Reply:
True, too much copy and paste.